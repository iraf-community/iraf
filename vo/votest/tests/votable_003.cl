#
#  Get a value from a VOTable cell

# Set the test description string.
votest.descr = "TABLES support: Get a value from a VOTable cell"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tabpar ("data$usno-b.xml", "I2", 10)
print ("cell value at (I2,10) = ", tabpar.value)

print ("\nHTTP URI:  " // s1)				# remote http URI
tabpar (s1, "I2", 10)
print ("cell value at (I2,10) = ", tabpar.value)

print ("\nFile URI:  " // s2)				# file URI
tabpar (s2, "I2", 10)
print ("cell value at (I2,10) = ", tabpar.value)

print ("\nFile URI:  " // s3)				# file URI
tabpar (s3, "I2", 10)
print ("cell value at (I2,10) = ", tabpar.value)
