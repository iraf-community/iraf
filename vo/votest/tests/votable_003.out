
Logical Path:  data$usno-b.xml
cell value at (I2,10) = 17.99

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
cell value at (I2,10) = 17.99

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
cell value at (I2,10) = 17.99

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
cell value at (I2,10) = 17.99
